src/Image.cc

```diff
@@ -420,6 +428,24 @@ Image::loadSurface() {
   if (isJPEG(buf)) return loadJPEG(stream);
 #endif
 
+// svg
+#ifdef HAVE_RSVG
+  // confirm svg using first 1000 chars
+  // if a very long comment precedes the root <svg> tag, isSVG returns false
+  uint8_t head[1000] = {0};
+  fseek(stream, 0 , SEEK_END);
```

Comment:
Would it be better to first call `isSVG` on the 5 bytes already read in `buf` and, if no `svg` bytes are found, then proceed to do `fseek` and `rewind`? It might save some ops, if most svg files got no xml or other comments at the beginning.